crates/biome_css_analyze/tests/specs/suspicious/noShorthandPropertyOverrides/invalid.css

```
@@ -35,3 +35,15 @@ a { font-size-adjust: 0.545; font: Verdana; }
 a { padding-left: 10px; padding: 20px; border-width: 20px; border: 1px solid black; }
 
 a { padding-left: 10px; padding-right: 10px; padding: 20px; }
+
+@keyframes fade { from { margin-left: 1px; margin: 0; } }
```

Stylelint (17.14.0)

[error] Overridden property "margin-left" by shorthand "margin" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)